test/algorithms/simulation/sidb/is_operational.cpp

```diff
@@ -36,6 +36,41 @@ TEST_CASE("SiQAD OR gate", "[is-operational]")
                                     operational_condition::REJECT_KINKS})
               .first == operational_status::NON_OPERATIONAL);
 
+    // determine if kinks induce layout to become non-operational.
+    const auto kink_induced_non_operational = is_kink_induced_non_operational(
+        lat, std::vector<tt>{create_or_tt()},
+        is_operational_params{
+            sidb_simulation_parameters{2, -0.28}, sidb_simulation_engine::QUICKEXACT,
+            bdl_input_iterator_params{detect_bdl_wires_params{1.5},
+                                      bdl_input_iterator_params::input_bdl_configuration::PERTURBER_ABSENCE_ENCODED},
+            operational_condition::REJECT_KINKS});
```

Comment:
You're currently creating these parameters over and over again. It would make sense to create them once as a variable or constant and then pass them to each call.